### PR TITLE
feat: schema_migrate SFN changes

### PR DESCRIPTION
**Pull request by ebezzi**

### CODE REVIEW

File: backend/layers/processing/schema_migration.py [hunk 1]
```diff
@@ -49,11 +50,18 @@ def gather_collections(self) -> List[Dict[str, str]]:
         # evaluate unpublished collections first, so that published versions are skipped if there is an active revision
         has_revision = []  # list of collections to skip if published with an active revision
         for collection in collections:
+
+            # TODO <testing code>
```
Comment (ebezzi) [inline backend/layers/processing/schema_migration.py:54]: This is useful for rdev testing. Purge before merging to main.

File: .happy/terraform/modules/schema_migration/main.tf [hunk 2]
```diff
@@ -251,15 +255,15 @@ resource aws_sfn_state_machine sfn_schema_migration {
                   },
                   {
                     "Name": "COLLECTION_ID",
-                    "Value": "$.collection_id"
+                    "Value.$": "$[0].Input.collection_id"
                   },
                   {
                     "Name": "CAN_PUBLISH",
-                    "Value": "$.can_publish"
+                    "Value": "true"
```
Comment (Bento007) [inline .happy/terraform/modules/schema_migration/main.tf:262]: This shouldn't be hardcoded

File: backend/layers/processing/requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-cellxgene-schema
+git+https://github.com/chanzuckerberg/single-cell-curation/#subdirectory=cellxgene_schema_cli
```
Comment (Bento007) [inline backend/layers/processing/requirements.txt:1]: Add a todo to revert this before merging to main

**Review by Bento007 — CHANGES_REQUESTED** [review]

File: .happy/terraform/modules/schema_migration/main.tf (hunk 2 above)
Comment (ebezzi) [inline .happy/terraform/modules/schema_migration/main.tf:262]: Couldn't find a reference to this in the code. Hardcoded to `true`.

**Pull request merged**